internal/graph/computed/computecheck.go

```diff
@@ -92,7 +99,7 @@ func computeCheck(ctx context.Context,
 
 	// Ensure that the number of resources IDs given to each dispatch call is not in excess of the maximum.
 	results := make(map[string]*v1.ResourceCheckResult, len(resourceIDs))
-	resultMetadatas := make(map[string]*v1.ResponseMeta, len(resourceIDs))
+	metadata := &v1.ResponseMeta{}
 
 	// TODO(jschorr): Should we make this run in parallel via the preloadedTaskRunner?
```

Comment:
unrelated to the fix in this PR, but it would appear the chunking is dispatching serially and blocking on the response, so for a wide relation this can increase the latency of the API request and potentially underutilize available resources.

The preloaded task runner is getting refactored in https://github.com/authzed/spicedb/pull/1453, so it would be a good opportunity to follow up and refactor the chunk function. If so I'd suggest opening an issue to keep track of this.